lib/typesense/config.rb

```diff
@@ -0,0 +1,30 @@
+module Typesense
+  module Config
+    def initiliaze
+      @client = nil
+    end
+
+    def configuration
+      @@configuration || raise(NotConfigured,
+                               "Please configure Typesense. Set Typesense.configuration = {application_id: 'YOUR_APPLICATION_ID', api_key: 'YOUR_API_KEY'}")
```

Comment:
@tharropoulos We don't have an `application_id`. Could you update this error message?